vpcmanager/src/main/java/com/futurewei/vpcmanager/comm/constants/Common.java

```diff
@@ -0,0 +1,709 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
If this is a generated code, they should be generated on the compile time, not stored in the src directory. Check how we handled it in the alcor/src directory.